cupy/core/_routines_statistics.pyx

```diff
@@ -348,6 +352,60 @@ cdef _mean = create_reduction_func(
      'out0 = a / _type_reduce(_in_ind.size() / _out_ind.size())', None))
 
 
+cdef _nanmean_preamble = '''
+#define ll long long
```

Comment:
Could you remove this line?
Please replace all `ll`s to `long long`, or use `typedef` instead of `#define`.

Reply:
Done